src/bin/cargo/cli.rs

```diff
@@ -569,7 +569,7 @@ Rust's package manager
     <cyan,bold>update</>      Update dependencies listed in Cargo.lock
     <cyan,bold>search</>      Search registry for crates
     <cyan,bold>publish</>     Package and upload this package to the registry
-    <cyan,bold>install</>     Install a Rust binary. Default location is $HOME/.cargo/bin
+    <cyan,bold>install</>     Install a Rust binary.
```

Comment:
```suggestion
    <cyan,bold>install</>     Install a Rust binary
```

Maybe we should remove this period here to align with other commands.